__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const db  = require("../db/connection");
+const app = require("../db/app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data")
+const request = require("supertest")
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("get/api/topics", () => {
+  test("Return topics", () => {
```

Comment:
This is good but could do with being slightly more descriptive. Something like "200 status code and returns an array of topics"